db/seeds.rb

```diff
@@ -11,11 +11,14 @@
 cat_reality = Category.create(title: 'Reality TV', description: 'reality tv')
 
```

Comment:
As long as your have `Faker` and `Fabricator` in your dev and test groups, you can use fabrication factories in your seeds file. Just so you know.